backend/config/corpora-api.yml

```diff
@@ -630,6 +667,30 @@ components:
           type: string
         detail:
           type: string
+    gene_sets:
```

Comment:
We have `gene_sets` and `geneset` on line 647. We should rename one of them to avoid confusion.

Reply:
how do you feel about gene_set_request and gene_set_response?